main.tf

```diff
@@ -4,7 +4,7 @@ terraform {
 }
 
 locals {
-  name = "new_account_iam_role_${random_string.id.result}"
+  name = "new-account-iam-role-${random_string.id.result}"
```

Comment:
This is a breaking change and will cause resource cycles. We missed the window to align the separator across these projects. It's not a terribly big deal if it's different from one project to the next, thankfully.

Reply:
ok